.github/workflows/check_new_kernel_configs.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Curious: why keep this script here and the other one inside `toolkit/scripts`?

Reply:
I can't really see a need for someone to use the check for a new config outside of the context for the workflow